src/ipahealthcheck/core/main.py

```diff
@@ -13,12 +13,18 @@
 from ipahealthcheck.core.plugin import Result, Results
 from ipahealthcheck.core.output import output_registry
 from ipahealthcheck.core import constants
-from ipahealthcheck.meta.services import ServiceCheck
-
 
 logging.basicConfig(format='%(message)s')
 logger = logging.getLogger()
 
+try:
+    from ipaserver.install.installutils import is_ipa_configured
+except ImportError:
+    logging.error("IPA server packages are not installed on this system.")
+    sys.exit(1)
+
+from ipahealthcheck.meta.services import ServiceCheck  # noqa: E402
```

Comment:
You can replace this with:

else:
    from ipahealthcheck.meta.services import ServiceCheck